src/test/java/io/reactivex/internal/operators/flowable/FlowableTakeUntilPredicateTest.java

```diff
@@ -29,9 +30,9 @@
 import io.reactivex.processors.PublishProcessor;
 import io.reactivex.subscribers.TestSubscriber;
 import io.reactivex.testsupport.*;
-;
 
-public class FlowableTakeUntilPredicateTest {
+
```

Comment:
Please remove this extra line as the style check fails.

Reply:
Thanks, I didn't understand the issue in the first place.